server/issue.go

```diff
@@ -169,9 +169,12 @@ func httpAPICreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (in
 		return http.StatusInternalServerError, errors.Errorf("Failed to create issue. %s", err.Error())
 	}
 
+	docsOrConnectLink := fmt.Sprintf("/plugins/%s%s", manifest.Id, routeToDocsOrConnect)
+	msg := fmt.Sprintf("Created Jira issue [%s](%s/browse/%s) by [mattermost-jira-plugin](%s)", created.Key, ji.GetURL(), created.Key, docsOrConnectLink)
```

Comment:
@aaronrothschild the link here should probably be `[Jira plugin]` - guidance?

Reply:
I'll implement this feedback as soon as we get guidance

Reply:
Let's leave as is, can change as a separate PR.